esp32-led-blink-sdk/diagram.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": 1,
+  "author": "Juraj Michálek",
```

Comment:
love the contribution, but could we exclude individual names? (in both json files)

Reply:
@rauhul  good point. Record updated.